Adds Daprd `--dapr-block-shutdown-duration` reference (#3893)
dapr/docs: daprdocs/content/en/reference/arguments-annotations-overview.md

| `--sentry-address` | `--sentry-address` | | not supported | Address for the [Sentry CA service]({{< ref sentry >}}) |
| `--version` | `--version` | `-v` | not supported | Prints the runtime version |
| `--dapr-graceful-shutdown-seconds` | not supported | | `dapr.io/graceful-shutdown-seconds` | Graceful shutdown duration in seconds for Dapr, the maximum duration before forced shutdown when waiting for all in-progress requests to complete. Defaults to `5`. If you are running in Kubernetes mode, this value should not be larger than the Kubernetes termination grace period, who's default value is `30`.|

> | `--dapr-block-shutdown-duration` | not supported | | `dapr.io/block-shutdown-duration` | Block shutdown duration, if set, will block the graceful shutdown proceedure as described above from starting until the given duration has elapsed or the application becomes unhealthy as configured through application health options. This is useful for applications which need to execute Dapr APIs during their own termination procudure. Any new invocations of any Dapr APIs will not be available to the application once the block has expired. Accepts [Go duration](https://pkg.go.dev/time#ParseDuration) string. |

Reviewer: ```suggestion
| `--dapr-block-shutdown-duration` | not supported | | `dapr.io/block-shutdown-duration` | Block shutdown duration, if set, blocks the graceful shutdown procedure (as described above) from starting until the given duration has elapsed or the application becomes unhealthy as configured through application health options. This is useful for applications that need to execute Dapr APIs during their own termination procedure. Any new invocations of any Dapr APIs are not available to the application once the block has expired. Accepts [Go duration](https://pkg.go.dev/time#ParseDuration) string. |
```
